### PR TITLE
accessibility issues fixes

### PR DESCRIPTION
**Pull request by biou**
see #375, this PR contains various accessibility fixes:

- cleaned html for invalid attributes
- keyboard navigation is now working correctly (focus management)
- fixed duplicate ids
- removed the aria-controls and aria-live attributes

### CODE REVIEW

**Comment by goblindegook** [comment]
Hi, @biou, and thanks for the PR. Could you resolve the merge conflicts and have a look at the tests, please? Some will need to be updated to account for your changes.

File: src/settings.ts [hunk 1]
```diff
@@ -36,6 +36,6 @@ export const DEFAULT_SETTINGS: Settings = {
   hoverDelay: 250,
   numberResetSelector: '',
   scope: '',
-  contentTemplate: `<aside class="littlefoot-footnote" id="fncontent:<% id %>" alt="Footnote <% number %>" aria-live="polite"><div class="${CLASS_WRAPPER}"><div class="${CLASS_CONTENT}" tabindex="0"><% content %></div></div><div class="${CLASS_TOOLTIP}"></div></aside>`,
```
Comment (goblindegook) [inline src/settings.ts:39]: Looks good, could you update the README as well, please?

The change also breaks a few tests if you could have a look at those.

File: src/dom/index.ts [hunk 2]
```diff
@@ -126,7 +126,7 @@ function prepareTemplateData(original: Original, idx: number): OriginalData {
     data: {
       id: `${idx + 1}`,
       number: idx + 1,
-      reference: original.referenceId,
```
Comment (goblindegook) [inline src/dom/index.ts:129]: This change breaks a few tests, which will need to be updated.

Comment (biou) [reply src/dom/index.ts:129]: The tests are updated now.

File: src/settings.ts (hunk 1 above)
Comment (biou) [reply src/settings.ts:39]: I updated the README. Regarding the changes in the templated, I had to remove also aria-describedby, for the same reason as aria-controls, i.e. most of the time the referenced element is not in the dom.

**Pull request merged**